packages/embed-react/README.md

```diff
@@ -160,6 +160,19 @@ and should not be conflated with transaction being declined or an error occuring
 }
 ```
 
+#### `transactionFailed`
```

Comment:
i think it might be worth calling out that this is called for 400/500s only. Where `transactionCreated` is called when the TX was accepted by Gr4vy but failed in the PSP.